package.json

```diff
@@ -1,5 +1,6 @@
 {
   "private": true,
+  "type": "module",
```

Comment:
Since we are adding `"type": "module"` to packages, `.js` files are treated as esm; therefore using `.mjs` or `.module.js` should not make a difference from the user perspective. I guess `.module.js` is more common in three ecosystem?

Reply:
changed to module.js in [2782638](https://github.com/pixiv/three-vrm/pull/1411/commits/27826382b8df2effd8ad34d9e289d2ac0c796113)

Reply:
much fewer diff!